<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>737</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Mono</family>
    <pointsize>14</pointsize>
    <italic>false</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Rexex</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/icons/icons/rexelLogo.png</normaloff>:/icons/icons/rexelLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="ServerSettings">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>1. Настройки сервера</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="IPLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Введите IP:port адрес сервера:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="IPEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>210</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>210</width>
             <height>25</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
            </font>
           </property>
           <property name="text">
            <string>127.0.0.1:2375</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string notr="true">***.***.***.***:*****</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="TryConnectButton">
           <property name="text">
            <string>Подключиться</string>
           </property>
           <property name="icon">
            <iconset theme="mail-send-receive"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ResetIPButton">
           <property name="text">
            <string>Сброс</string>
           </property>
           <property name="icon">
            <iconset theme="application-exit"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ConnectionStatus">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="CalcSettings">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>2. Настройки вычислений</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="MainFileChoose">
         <item>
          <widget class="QLabel" name="MainFileLabel">
           <property name="text">
            <string>Исполняемый файл: -</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ChooseMainFileButton">
           <property name="maximumSize">
            <size>
             <width>95</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Выбрать</string>
           </property>
           <property name="icon">
            <iconset theme="folder-new"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="MainFileResetButton">
           <property name="minimumSize">
            <size>
             <width>104</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>104</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Сбросить</string>
           </property>
           <property name="icon">
            <iconset theme="application-exit"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="ReqChoose">
         <item>
          <widget class="QLabel" name="ReqLabel">
           <property name="text">
            <string>Файл requirements: -</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ReqChooseButton">
           <property name="maximumSize">
            <size>
             <width>95</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Выбрать</string>
           </property>
           <property name="icon">
            <iconset theme="folder-new"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ReqResetButton">
           <property name="maximumSize">
            <size>
             <width>104</width>
             <height>27</height>
            </size>
           </property>
           <property name="text">
            <string>Сбросить</string>
           </property>
           <property name="icon">
            <iconset theme="application-exit"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="listFileLabel">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Выбор дополнительных файлов:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listFileWidget">
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="AddRemovw">
         <item>
          <widget class="QPushButton" name="AddFileButton">
           <property name="text">
            <string>Добавить</string>
           </property>
           <property name="icon">
            <iconset theme="list-add"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveButton">
           <property name="text">
            <string>Убрать</string>
           </property>
           <property name="icon">
            <iconset theme="application-exit"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="RunButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>35</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Запустить</string>
      </property>
      <property name="icon">
       <iconset theme="media-playback-start"/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="Result">
      <property name="accessibleName">
       <string/>
      </property>
      <property name="title">
       <string>3. Результат</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="usesScrollButtons">
          <bool>true</bool>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <property name="tabBarAutoHide">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="consoleOutput">
          <property name="accessibleName">
           <string/>
          </property>
          <attribute name="title">
           <string>Вывод консоли</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTextEdit" name="serverOutput">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="files">
          <attribute name="title">
           <string>Файлы</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="1">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QListWidget" name="DirDiffList">
               <property name="alternatingRowColors">
                <bool>false</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
               </property>
               <property name="isWrapping" stdset="0">
                <bool>false</bool>
               </property>
               <property name="viewMode">
                <enum>QListView::ViewMode::ListMode</enum>
               </property>
               <property name="selectionRectVisible">
                <bool>true</bool>
               </property>
               <property name="sortingEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="InstallResponseButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Загрузить</string>
               </property>
               <property name="icon">
                <iconset theme="document-save"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>IPEdit</tabstop>
  <tabstop>TryConnectButton</tabstop>
  <tabstop>ResetIPButton</tabstop>
  <tabstop>ChooseMainFileButton</tabstop>
  <tabstop>MainFileResetButton</tabstop>
  <tabstop>ReqChooseButton</tabstop>
  <tabstop>ReqResetButton</tabstop>
  <tabstop>listFileWidget</tabstop>
  <tabstop>AddFileButton</tabstop>
  <tabstop>RemoveButton</tabstop>
  <tabstop>RunButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>serverOutput</tabstop>
  <tabstop>DirDiffList</tabstop>
  <tabstop>InstallResponseButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
 <slots>
  <slot>IPEdit()</slot>
  <slot>TryConnect()</slot>
  <slot>ResetIP()</slot>
  <slot>AddMainFile()</slot>
  <slot>ResetMainFile()</slot>
 </slots>
</ui>
